app/src/utils/test/db/profile.ts

```diff
@@ -96,13 +96,22 @@ export const intakeProfiles: GetProfileApiResponse[] = [
           },
           {
             id: faker.string.numeric(4),
-            title: 'Pets',
+            title: 'Do you have any pets?',
```

Comment:
I believe it should be "Do you have any pets in your care?

Reply:
Thanks for catching this. I updated it.